### ISSUE TITLE
[Solution] CS:Go segfault at startup

### ISSUE THREAD

**Issue by Aaron1011** [ValveSoftware/steam-for-linux#3917]
I'm posting my solution to this issue here, in the hopes that someone finds it useful.

TL;DR: Remove `~/.local/share/Steam/ubuntu12_32/steam-runtime/i386/lib/i386-linux-gnu/libudev.so.0` to fix the issue.

When trying to launch CS:GO on Ubuntu 14.10, a segfault occurred (which was caught by Steam) after the 'Preparing to Launch CS:Go' screen appeared. Reinstalling CS:GO and Steam didn't fix the issue.

By editing the `csgo_linux` script, I was able to use GDB to trace down the segfault to the `libudev.so.0' library included in the steam runtime. Thinking this might have been an issue with the version shipping with Steam, I removed it. This fixed the issue, and CS:GO launched with no issues.

For reference, here's the backtrace from GDB (`/home/aaron` is my home directory, and `/external/Steam` is the root of the Steam library folder):

```
Program received signal SIGSEGV, Segmentation fault.
0xf7df5d06 in ?? () from /lib/i386-linux-gnu/libc.so.6
(gdb) bt
#0  0xf7df5d06 in ?? () from /lib/i386-linux-gnu/libc.so.6
#1  0xf794e46c in ?? () from /lib/i386-linux-gnu/libudev.so.1
#2  0xf7953212 in udev_monitor_filter_update () from /lib/i386-linux-gnu/libudev.so.1
#3  0xf4c4913e in udev_monitor_enable_receiving () from /home/aaron/.local/share/Steam/ubuntu12_32/steam-runtime/i386/lib/i386-linux-gnu/libudev.so.0
#4  0xf60ef70f in ?? () from /external/Steam/steamapps/common/Counter-Strike Global Offensive/bin/libSDL2-2.0.so.0
#5  0xf60daa67 in ?? () from /external/Steam/steamapps/common/Counter-Strike Global Offensive/bin/libSDL2-2.0.so.0
#6  0xf6051874 in SDL_InitSubSystem () from /external/Steam/steamapps/common/Counter-Strike Global Offensive/bin/libSDL2-2.0.so.0
#7  0xf3ad4267 in ?? () from /external/Steam/steamapps/common/Counter-Strike Global Offensive/bin/inputsystem_client.so
#8  0xf3ad1c08 in ?? () from /external/Steam/steamapps/common/Counter-Strike Global Offensive/bin/inputsystem_client.so
#9  0xf66ef776 in ?? () from bin/launcher_client.so
#10 0xf66f0660 in ?? () from bin/launcher_client.so
#11 0xf66f06bd in ?? () from bin/launcher_client.so
#12 0xf66f06f0 in ?? () from bin/launcher_client.so
#13 0xf66d26c9 in LauncherMain () from bin/launcher_client.so
#14 0x08048544 in main ()
(gdb) continInstalling breakpad exception handler for appid(steam)/version(1433441724)
```

**Comment by kisak-valve** [comment]
Hello @Aaron1011, are you still experiencing this issue on an up to date system?

**Comment by kisak-valve** [comment]
Closing pending feedback.
